commands/dns-records.go

```diff
@@ -31,27 +82,31 @@ func (cmd *DNSRecords) Commands() []cli.Command {
 
 // Run executes the `rig dns-records` command
 func (cmd *DNSRecords) Run(c *cli.Context) error {
+	// Don't require rig to be started to remove records
+	if c.Bool("remove") {
+		if c.Bool("save") {
+			return cmd.Failure("--remove and --save are mutually exclusive", "COMMAND-ERROR", 13)
+		}
+		// TODO The VM might have to be up for Windows
+		return cmd.removeDNSRecords()
+	}
 
 	records, err := cmd.LoadRecords()
 	if err != nil {
 		return cmd.Failure(err.Error(), "COMMAND-ERROR", 13)
 	}
 
-	for _, record := range records {
-		for _, ip := range record["IPs"].([]interface{}) {
-			fmt.Printf("%s\t%s.%s.%s\n", ip, record["Name"], record["Image"], "vm")
-			// attach any aliases too
-			for _, a := range record["Aliases"].([]interface{}) {
-				fmt.Printf("%s\t%s\n", ip, a)
-			}
-		}
+	if c.Bool("save") {
+		return cmd.saveDNSRecords(records)
 	}
 
+	printDNSRecords(records)
```

Comment:
if you do a `save` or `remove` you likely don't want the records printed out too so maybe we only print the records if `save` or `remove` have not been specified

Reply:
They always return if they do a save or remove.  So it just falls through to the default if they don't specify either.

Reply:
OH yeah. ha. You got it.